RiotSharp/LeagueEndpoint/LeaguePosition.cs

```diff
@@ -5,38 +5,60 @@ namespace RiotSharp.LeagueEndpoint
     /// <summary>
     /// Team or summoner in a league (League API).
     /// </summary>
-    public class LeagueEntry
+    public class LeaguePosition
     {
-        internal LeagueEntry() { }
+        internal LeaguePosition() { }
 
         /// <summary>
+        /// The name of the league of the participant.
+        /// Only when it's called from the GetLeaguePositions()
+        /// </summary>
+        [JsonProperty("leagueName")]
+        public string LeagueName { get; set; }
+
+        /// <summary>
+        /// The queue type of the league.
+        /// Only for the GetLeaguePositions() -> don't exist when it's an entry from a League
+        /// as there is already the Queue property in this case.
+        /// </summary>
+        [JsonProperty("queueType")]
+        public string QueueType { get; set; }
+        
+        /// <summary>
         /// The league division of the participant.
         /// </summary>
-        [JsonProperty("division")]
+        [JsonProperty("rank")]
         public string Division { get; set; }
 
+        ///<summary>
+        /// The league tier of the participant.
+        /// Only when it's called from the GetLeaguePositions()
+        /// </summary>
+        [JsonProperty("tier")]
+        public string Tier { get; set; }
```

Comment:
I don't see tier in the `LeaguePositionDTO` in https://developer.riotgames.com/api-methods/#league-v3/GET_getAllLeaguePositionsForSummoner same goes for `queueType` and `leagueName`

Reply:
When i'm doing a request I'm having them : 
https://i.imgur.com/EzBYdGv.png
I think RIOT forgot to add it in the doc 🤷‍♂️

Reply:
welp thx rito :smile:

Reply:
I would not be surprise if this error is present for other parts of the API 🤔

Reply:
I made a ticket to warn them. Now it's on their hands 🤷‍♂️

Reply:
Still no news from Riot Developer Support :/

Reply:
I pointed out the difference in the Riot Api Discord server and most of the people was already aware of it, but Rito doesn't really care.

Reply:
🤷‍♂️